src/tests/async.test.ts

```diff
@@ -207,6 +207,25 @@ describe("async module", () => {
       assert.equal(result, 'hello')
     })
   })
+
+  describe('_.tryit function', () => {
+    test('returns error when error is thrown', async () => {
+      const [err, result] = await _.tryit(async () => {
+        throw new Error('not good enough')
+      })()
+      assert.isNull(result)
+      assert.isNotNull(err)
+      assert.equal(err.message, 'not good enough')
+    })
+    test('returns result when no error is thrown', async () => {
+      const [err, result] = await _.tryit(async () => {
+        return 'hello'
+      })()
+      assert.isNull(err)
+      assert.isNotNull(result)
+      assert.equal(result, 'hello')
+    })
+  })
```

Comment:
The `tryit` function is already covered by the tests. Adding an aliased export wont effect the test coverage so we can remove these.

Reply:
but the test coverage fails

Reply:
![Screen Shot 2022-08-28 at 11 01 34 AM](https://user-images.githubusercontent.com/23265119/187066252-30656310-0540-49d1-86d4-913a59cadb30.png)

Reply:
The method is tested but the alias is not tested. It is a little bit redundant but needed for coverage

Reply:
Ah gotcha, I think all we need to do is import it and it should get covered. Can you try adding a one-line test like this?
```ts
test('alias exists', () => { assert.notNull(tryit) } 
```

Reply:
Sure